Common/src/config_structure.cpp

```diff
@@ -757,8 +757,8 @@ void CConfig::SetConfig_Options(unsigned short val_iZone, unsigned short val_nZo
   addStringListOption("MARKER_FLUID_INTERFACE", nMarker_Fluid_InterfaceBound, Marker_Fluid_InterfaceBound);
   /*!\brief MARKER_INTERFACE\n DESCRIPTION: Zone interface boundary marker(s) \ingroup Config*/
   addStringListOption("MARKER_INTERFACE", nMarker_InterfaceBound, Marker_InterfaceBound);
-  /*!\brief MARKER_FSI_INTERFACE \n DESCRIPTION: FSI interface boundary marker(s) \ingroup Config*/
-  addStringListOption("MARKER_FSI_INTERFACE", nMarker_FSIinterface, Marker_FSIinterface);
+  /*!\brief MARKER_FSI_INTERFACE \n DESCRIPTION: ZONE interface boundary marker(s) \ingroup Config*/
+  addStringListOption("MARKER_ZONE_INTERFACE", nMarker_ZoneInterface, Marker_ZoneInterface);
   /*!\brief MARKER_DIRICHLET  \n DESCRIPTION: Dirichlet boundary marker(s) \ingroup Config*/
```

Comment:
I see your point which is fair, but I don't think that moving to a general BC is going to be straight forward, perhaps it would be better to define a new BC to treat such hybrid cases.
We should carefully think about it to get to manage all the possibilities, and maybe we can have a chat in the future (cause I'm not really into structural stuff so I may not see the bigger picture), but right now I would keep things separated.

Sorry, I was unclear, I meant that the method that corresponds to the fluid_interface marker is included in the solver_direct_mean.cpp.